src/Core/Components/Radio/FluentRadio.razor.cs

```diff
@@ -0,0 +1,81 @@
+// ------------------------------------------------------------------------
+// MIT License - Copyright (c) Microsoft Corporation. All rights reserved.
+// ------------------------------------------------------------------------
+
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.AspNetCore.Components;
+using Microsoft.FluentUI.AspNetCore.Components.Extensions;
+using Microsoft.FluentUI.AspNetCore.Components.Utilities;
+
+namespace Microsoft.FluentUI.AspNetCore.Components;
+
+/// <summary>
+/// A Fluent Radio button component.
+/// </summary>
+/// <typeparam name="TValue">The type for the value of the radio button</typeparam>
+public partial class FluentRadio<[DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.All)] TValue> : FluentInputBase<TValue>, IFluentField
+{
+    bool _trueValueToggle;
+
+    internal FluentRadioContext? Context { get; private set; }
+
+    /// <summary />
+    public FluentRadio()
+    {
+        Id = Identifier.NewId();
+    }
+
+    [CascadingParameter]
+    private FluentRadioContext? CascadedContext { get; set; }
+
+    /// <summary>
+    /// Gets or sets a value indicating whether the element is checked.
+    /// </summary>
+    [Parameter]
+    public bool? Checked { get; set; }
```

Comment:
I don't understand in which conditions this parameter will be used (outside of a Group). And in this case, we should create a bindable property. But for me, we could remove it and "force" the dev to use an Radio only inside a group.

Reply:
It is used to set the initial checked state of a radio item in a group. We do not want to make this bindable as the binding takes place on the group level (with the `Value` parameter)

Reply:
Weird. And what if the first option `FluentOption.Checked=true` and the `FluentOptionGroup.Value="Second"` ?
Why not to use only the bindable FluentOptionGroupd value?